src/commands/npm/package/release.ts

```diff
@@ -71,33 +72,22 @@ export default class Release extends SfCommand<ReleaseResult> {
     const deps = verifyDependencies(flags);
     if (deps.failures > 0) {
       const errType = 'MissingDependencies';
-      const missing = deps.results.filter((d) => d.passed === false).map((d) => d.message);
+      const missing = deps.results
+        .filter((d) => d.passed === false)
+        .map((d) => d.message)
+        .filter(isString);
       throw new SfError(messages.getMessage(errType), errType, missing);
     }
 
     const pkg = await PackageRepo.create({
       ux: new Ux({ jsonEnabled: this.jsonEnabled() }),
       useprerelease: flags.prerelease,
     });
-    if (!pkg.shouldBePublished) {
```

Comment:
we've taken out all the publishing decisions now that we npm-publish from GH Release